### DISCUSSION TITLE
[OP Stack Deployment] Issue with pnpm build failing

### DISCUSSION THREAD

**Discussion by yohanelly95** [ethereum-optimism/developers#382] [Deploying the OP Stack] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

running pnpm build fails with an error one of the packages cannot be built. 
specifically the 
 - nx run @eth-optimism/chain-mon:build

### Logs

Ran target build for 8 projects (12s)

   ✔  7/8 succeeded [7 read from cache]

   ✖  1/8 targets failed, including the following:

      - nx run @eth-optimism/chain-mon:build

View structured, searchable error logs at https://nx.app/runs/weE5q0byX5

**Comment by Chomtana** [comment]
Which branch do you use? Please use op-contracts/v1.3.0 or tutorials/chain

**Reply by yohanelly95** [reply]
I used the latest release `tags/v1.7.3`. For testnet and production should we use `op-contracts/v1.3.0` ?

**Reply by Chomtana** [reply, marked as answer]
Yes, you should use `op-contracts/v1.3.0` to deploy the contract

And then switch back to `v1.7.3` (Now `v1.7.5`) for building go binaries (op-node, op-batcher, op-proposer)

But you probably don't need to do pnpm build on `v1.7.5`

Just make op-node op-batcher op-proposer
